gradle.properties

```diff
@@ -7,7 +7,7 @@ cordaProductVersion = 5.0.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 132
+cordaApiRevision = 133
```

Comment:
Bit late, but this probably didn't need to be a version bump as I don't think it's a breaking change

Reply:
I guess I don't fully understand how this works. Does `corda-runtime-os` pick up the latest from `corda-api` regardless of what you specify here?

Reply:
Or at least the latest for the matching `cordaApiRevision`?

Reply:
The runtime-os references the API using the wildcard, like cordaApiVersion=5.0.0.131-beta+ which means that the build will pick the latest version as the beta version is composed using the timestamp suffix